migrations/versions/723c650ca61c_add_user_and_subscription_models.py

```diff
@@ -0,0 +1,42 @@
+"""Add User and Subscription models
+
+Revision ID: 723c650ca61c
+Revises: 1add42f9be06
+Create Date: 2022-12-21 14:43:28.587026
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '723c650ca61c'
+down_revision = '1add42f9be06'
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('user',
+    sa.Column('name', sa.String(), nullable=False),
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.PrimaryKeyConstraint('id')
+    )
+    op.create_table('subscription',
+    sa.Column('user_id', sa.Integer(), nullable=False),
+    sa.Column('zone_id', sa.Integer(), nullable=False),
```

Comment:
Can we add uniq constraint so there would be no doubled subscriptions for same user — same zone?

Reply:
was going to check the uniqueness of the subscription while adding a new one, not sure how to constraint it by db